roles/validate/tasks/sub_main.yml

```diff
@@ -85,6 +85,8 @@
     inventory_hostname: "{{ inventory_hostname }}"
     hostvars: "{{ hostvars }}"
     model_data: "{{ model_data['data'] }}"
+    default_values: "{{ defaults }}"
```

Comment:
The way we manage defaults is changing.  We will need to refactor this a bit to account for that.  Changes introducing custom defaults

https://github.com/netascode/ansible-dc-vxlan/pull/197